ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -17,11 +18,17 @@ import I18n from "../../i18n";
 import { resetFirstOnboarding } from "../../store/actions/onboarding";
 import { deletePin } from "../../utils/keychain";
 import { resetPreferences } from "../../store/actions/persistedPreferences";
+import { useHeaderSecondLevel } from "../../hooks/useHeaderSecondLevel";
+import { IOScrollViewWithLargeHeader } from "../../components/ui/IOScrollViewWithLargeHeader";
 
 const consecutiveTapRequired = 4;
 const RESET_COUNTER_TIMEOUT = 2000 as Millisecond;
 
 const ProfileMainScreen = () => {
+  useHeaderSecondLevel({
+    title: ""
```

Comment:
**question**: Is this empty string correct?

Reply:
Nice catch, it is correct as we don't want to show a header title. I'm not sure why this interface declares this parameter as required as in a lot of cases it won't be shown anyway. It only appears when scrolling over the internal view. The component has been ported from `io-app`.